diff --git a/trunk/LUFA/Drivers/USB/Class/Common/HIDClassCommon.h b/trunk/LUFA/Drivers/USB/Class/Common/HIDClassCommon.h
index c3fe3ed..0ccd56a 100644
--- a/LUFA/LUFA/Drivers/USB/Class/Common/HIDClassCommon.h
+++ b/LUFA/LUFA/Drivers/USB/Class/Common/HIDClassCommon.h
@@ -626,6 +626,13 @@
 			uint8_t Button; /**< Button mask for currently pressed buttons in the mouse. */
 			int8_t  X; /**< Current delta X movement of the mouse. */
 			int8_t  Y; /**< Current delta Y movement on the mouse. */
+
+			#ifdef MOUSE_HAS_SCROLL_WHEELS
+			// also adjust descriptors
+			int8_t  V; /**< Vertical wheel */
+			int8_t  H; /**< Horizontal wheel */
+			#endif
+
 		} ATTR_PACKED USB_MouseReport_Data_t;
 
 		/** \brief Standard HID Boot Protocol Keyboard Report.
